ibis/expr/types/relations.py

```diff
@@ -1334,6 +1334,66 @@ def sql(self, query: str) -> ir.Table:
         )
         return op.to_expr()
 
+    def histogram(
+        self,
+        column: str,
+        nbins: int | None = None,
+        binwidth: float | None = None,
+        base: float | None = None,
+        aux_hash: str | None = None,
+        eps: float = 1e-13,
+    ):
+        """Compute a histogram with fixed width bins.
+
+        Parameters
+        ----------
+        column
+            Column to bucket
+        nbins
+            If supplied, will be used to compute the binwidth
+        binwidth
+            If not supplied, computed from the data (actual max and min values)
+        base
+            Histogram base
+        aux_hash
```

Comment:
Since this is a breaking change moving this method, perhaps we should also take the chance to rename this keyword? This is just a column name suffix, it doesn't actually have to be a hash. Perhaps `suffix`, `col_suffix`, `bin_suffix` (I like `suffix` best, since it's similar to `suffixes` in `join`)?

Reply:
Yeah, `suffix` sounds good to me

Reply:
OTOH, do we even need this level of optionality here?